bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
Ugh, yeah, that is bad.

I'm not sure how to best test this. We could do clone with *only* unshare USERNS, and it that fails even though we're unprivileged (for whatever reason) we can assume its not working?